prisma/schema.prisma

```diff
@@ -6,3 +6,30 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+enum Role {
+  Admin
+  SuperAdmin
+  Customer
+  Trainer
+}
+
+enum UserStatusEnum {
+  Active
+  Pending
+}
+
+model User {
+  id          Int             @id @default(autoincrement())
+  firstName   String          @db.VarChar(255)
```

Comment:
why do we need such large numbers of characters in firstName, lastName, password, phone fields?

Reply:
For password field it's necessary because we keep password in hashed format.
For other fields to avoid interacting with the user model in the future.